cypress/integration/pages/topicPage/index.js

```diff
@@ -17,9 +18,12 @@ Object.keys(config)
       describe(`${pageType} - ${currentPath}`, () => {
         before(() => {
           Cypress.env('currentPath', currentPath);
-
-          const newPath = `${currentPath}${overrideRendererOnTest()}`;
-
+          let newPath;
```

Comment:
You could make this a const again and use a ternary so that instead of checking where the value changes (in this case, 2 places) I know that the value never changes

Reply:
Yes, nice, I'll do that 👍